package.json

```diff
@@ -67,7 +66,7 @@
     "karma": "^1.1.1",
     "karma-browserify": "^5.0.1",
     "karma-cli": "^1.0.0",
-    "karma-coverage": "^1.0.0",
+    "karma-coverage": "1.1.2",
```

Comment:
I know this is a dev dependency, but seeing one of locked version is still making me think there was a reason why? Is there a reason why this is locked?